Integrations/integration-OpsGenie.yml

```diff
@@ -0,0 +1,141 @@
+description: Get current on-call assignments and users info
+detaileddescription: OpsGenie integration allows querying for the current on-call
+  users for a given Schedule. The integration requires an API key, that can
+  be acquired from the OpsGenie interface, under Integrations / API.
+configuration:
+- display: API Key
+  name: apiKey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: User system proxy configuration
```

Comment:
insecure?
+ usually the order is
server url
api key
proxy and insecure

Reply:
it's a cloud service, so no need insecure. will change order.